README.md

```diff
@@ -3,3 +3,12 @@
 ![GitHub](https://img.shields.io/github/license/scylladb/sstable-tools)
 
 
+# contributing
+
+
+```
+pip install -r requirement-dev.txt
+pre-commit install
+
+# now each commit you do would fixed by black
```

Comment:
I don't understand this sentence.

Reply:
I was me trying to explain yelp's pre-commit, with small success :sob: 

https://pre-commit.com/

I'll remove this line